legend-engine-xt-serviceStore-pure/src/main/resources/core_servicestore/protocols/pure/vX_X_X/models/metamodel_serviceStore.pure

```diff
@@ -93,6 +94,17 @@ Class <<typemodifiers.abstract>> meta::protocols::pure::vX_X_X::metamodel::store
    _type    : String[1];
 }
 
+Class meta::protocols::pure::vX_X_X::metamodel::store::service::SimpleHttpSecurityScheme extends meta::protocols::pure::vX_X_X::metamodel::store::service::SecurityScheme
```

Comment:
why we have simple in name? also maybe we need similar property addition as in metamodel